resources/js/Pages/Auth/UserStatus.vue

```diff
@@ -0,0 +1,25 @@
+<script>
+export default {
+  computed: {
+    isAuthenticated() {
+      return this.$page.props.auth.loggedIn
+    },
+    user() {
```

Comment:
```suggestion
<script>
export default {
  computed: {
    isAuthenticated() {
      return this.$page.props.auth.loggedIn
    },
    user() {
```
I would create page for logged users only, login page should be for not logged users.